### PR TITLE
Add initial testing documentation and samples

### PR DESCRIPTION
**Pull request by nonword**
Add basic instructions for running Jmeter tests and samples projects for discovery-api, DFE, and sierra.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+
+## Usage
+
+See the [discovery-api](./discovery-api) and [sierra-api](./sierra-api) sample projects.
+
+### Create some paths to test
+
+The generic testing script depends on a CSV containing a bunch of request paths to test at random. Ideally it should contain thousands of paths that are representative of typical usage of the API under testing.
+
+See [discovery-api sample paths](./disocvery-api/sample-paths.csv).
```
Comment (charmingduchess) [inline README.md]: typo

Comment (nonword) [reply README.md]: Fixed

**Comment by charmingduchess** [comment]
I'm working through the instructions in the readme, and I'm getting an error trying to access `random-api-endpoints.jmx`. 
Getting this error: `An error occurred: The file /Users/verakahn/Documents/utils/lsp-stress-testing/discovery-api/../random-api-endpoints.jmx doesn't exist or can't be opened`
Am I supposed to make one? I made one in the root, cd'd into discovery-api, and ran the HEAP script. Which resulted in this error:
`Error in NonGUIDriver com.thoughtworks.xstream.io.StreamException: 
An error occurred: Error in NonGUIDriver `

**Comment by nonword** [comment]
I updated the README to clarify: Better to run it via the sample discovery-api script.

**Comment by charmingduchess** [comment]
Still having trouble getting code running, but I can suggest for now that you include vendor directories in `.gitignore`

**Comment by charmingduchess** [comment]
Got it to run! I had some issues installing `httparty`, but I think it was just something local. I had to sudo to make it go. 
Looks good to me, just a couple notes:
One more docs suggestion which is to include a rough estimate of how long the script will take to run. I hit go without any idea and then worried that I had kicked off something endless...
Also, I was unable to run the `generate-api-paths.rb` script with the comment present on line 30.

**Review by charmingduchess — APPROVED** [review]

**Pull request merged**